SwiftVoiceQuickstart/ViewController.swift

```diff
@@ -28,13 +28,13 @@ class ViewController: UIViewController, TVONotificationDelegate, TVOCallDelegate
     @IBOutlet weak var muteSwitch: UISwitch!
     @IBOutlet weak var speakerSwitch: UISwitch!
 
-    var incomingPushCompletionCallback: (()->Swift.Void?)? = nil
+    var incomingPushCompletionCallback: (() -> Void)? = nil
```

Comment:
I think you can remove `= nil` from this and the similar lines below. It should be inferred for any optional var.